src/Management/src/EndpointBase/Info/EpochSecondsDateTimeConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Steeltoe.Management.Endpoint.Info
+{
+    public class EpochSecondsDateTimeConverter : JsonConverter<DateTime>
+    {
+        private static readonly DateTime _baseTime = new (1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options) => DateTime.Parse(reader.GetString());
```

Comment:
I could be wrong, but I don't think this would actually get used (by us)? Would it be better to throw a `NotImplementedException`?

Reply:
No not a notimplementedException, typically the converter is absent, the conversion simply doesn't happen. But if its there and input is formatted it will throw a formatexception which is fine in this case .